### PR TITLE
goose version command fix

### PR DESCRIPTION
**Pull request by elenazherdeva**
Solving the issue https://github.com/square/goose/issues/29

- Created an entry point for `goose` (`goose-ai` package)
- Fixed the issue where modules weren’t imported correctly during the `uv run goose version`

Corresponding PR in exchange:
https://github.com/square/exchange/pull/16

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,9 @@ default = "goose.profile:default_profile"
 [project.entry-points."goose.command"]
 file = "goose.command.file:FileCommand"
 
+[project.entry-points."metadata.plugins"]
+goose-ai = ""
```
Comment (zakiali) [inline pyproject.toml]: Take it doing something like package_name="goose-ai" didn't work?

**Comment by zakiali** [comment]
Mind adding a description of this into the pr description?

File: pyproject.toml (hunk 1 above)
Comment (lukealvoeiro) [reply pyproject.toml]: I just realized the exports probably would overwrite each other for that case

**Review by lukealvoeiro — COMMENTED** [review]
LGTM, pending writing a PR description for posterity & and including the new version of exchange (that exports the correct group)

File: pyproject.toml (hunk 1 above)
Comment (zakiali) [reply pyproject.toml]: Yes, I think you’re right! Good point

**Review by zakiali — APPROVED** [review]
lgtm!

**Pull request merged**